src/utilities/isObject.ts

```diff
@@ -0,0 +1,8 @@
+import {TypeOf} from '../types';
+
+function isObject(value: any) {
+  const type = typeof value;
+  return value != null && (type === TypeOf.Object || type === TypeOf.Function);
+}
+
+export default isObject;
```

Comment:
It looks like all of our other files in src/utilities export named functions, and don't export any default. Should we keep consistent with that behaviour here too?

Reply:
That's fair, keeping it consistent would be nice. Do you think for utilities that will only have one export they should export a default, or should we continue with the way things are now?

Reply:
I don't think there's much difference between a single named export and a default. As theres no inherently superior solution I reckon stick with named export so we are consistent - that way the next time someone needs to add a utility they can see we have a convention and can follow it instead of having to ask "hey we do this in two ways, which one should I follow".

Then if we want to revisit this in the future we can change all utilities at once.